RadixWallet/Features/AccountRecoveryScan/Children/AccountRecoveryScanInProgress/AccountRecoveryScanInProgress.swift

```diff
@@ -320,6 +327,9 @@ extension AccountRecoveryScanInProgress {
 					)
 				)
 			)
+		} catch: { error, send in
+			loggerGlobal.error("Failed to scan network, error: \(error)")
+			await send(.delegate(.failed))
```

Comment:
another important fix: this ensure that we are not stuck in a loading state, but rather we dismiss the reducer...